multinodeNVL/nvbandwidth/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM nvcr.io/nvidia/cuda:12.5.0-devel-ubuntu22.04 AS builder
```

Comment:
Nit: License header.

Reply:
We should also make the cuda version here an argument so that we can easily update to a newer one without having to change this file

Reply:
yes. need the license and the cuda version as ARG

Reply:
@klueska on other projects we have moved to defining the versions in the `FROM` line directly since this allows us to leverage dependabot automation to update our base images instead of doing this manually.